hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3ARequesterPays.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.fs.s3a;
+
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.contract.s3a.S3AContract;
+import org.apache.hadoop.fs.statistics.StreamStatisticNames;
+import org.junit.Test;
```

Comment:
nit: junit (and other non org.apache.*) imports should go into their own block above the apache ones. Yes, all our existing code isn't always so good ,no, we can't fix that without ruining the lives of anyone cherrypicking. But we try...

Reply:
Sounds good.

If there is a configuration file I can import for IntelliJ, let me know. I already imported the Checkstyle but I don't think it applies here. In the meantime, I added the import order to my configuration.

Reply:
i don't think there is, sorry.  let me add one as a gist and link

here you go; i can do an xml one if that is preferable
https://gist.github.com/steveloughran/817dd90e0f1775ce2b6f24684dfb078c

Reply:
This works great, I'll share it with the team